src/test/scala/za/co/absa/hyperdrive/trigger/scheduler/executors/shell/ShellExecutorTest.scala

```diff
@@ -98,7 +99,12 @@ class ShellExecutorTest extends FlatSpec with Matchers with BeforeAndAfterAll wi
     when(updateJobStub.apply(any[JobInstance])).thenReturn(Future.successful((): Unit))
     val testInput = testJobInstance.copy(jobStatus = Submitting)
 
-    Await.result(ShellExecutor.execute(testInput, updateJobStub.apply), Duration(120, TimeUnit.SECONDS))
+    val shellParameters: ShellParameters = testInput.jobParameters match {
```

Comment:
Since it's a test, it would be ok for me to just use `asInstanceOf`

Reply:
It is done this way, I will not rewrite it to asInstanceOf